code/game/machinery/computer/law.dm

```diff
@@ -1,3 +1,5 @@
+#define cooldown_delay = 60 SECONDS
```

Comment:
```suggestion
#define EMAG_COOLDOWN 60 SECONDS
```
No need for "=" and i think it would be better to name it like this.
At the end of the "aiupload/attack_ghost" do #undef EMAG_COOLDOWN too, like the example i sent you before.